File: 240718/특정 숫자 도달하기/reaching-specific-number.py
# 배열을 구현하여 주어진 수를 입력받습니다.
arr = list(map(int, input().split()))

sum_val = 0
cnt = 0

# 10개의 정수 중 250 이상의 수가 나올 때 까지의 수의 합계와 개수를 구합니다.
for e in arr:
	if e >= 250:
		break
	sum_val = sum_val + e
	cnt += 1

# 평균을 구합니다.
avg = sum_val / cnt
 
# 출력
print(f"{sum_val} {avg:.1f}")

'''
--.pop : 마지막 원소 삭제
--.append : 하나씩 넣기
'''

File: 241129/특정 숫자 도달하기/reaching-specific-number.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int arr[10];
    int cnt = 0, sum_val = 0;

    for (int i=0; i<10; i++){
        scanf("%d ", &arr[i]);
        cnt+=1;
        sum_val+=arr[i];
        
        if(arr[i] >= 250) {
			sum_val = sum_val - arr[i];
			cnt = cnt -1;
            break;
        }
    }
	printf("%d %.1lf", sum_val, (double)sum_val/cnt);

    return 0;
}